cypress/e2e/login_management/studentFirstLogin.feature

```diff
@@ -0,0 +1,37 @@
+@pr
+@stable_test
+Feature: First Login - Login of new student user by SVS
+
+    As a user, I want to login the first time with username and password into SVS
+
+    Scenario Outline: User makes a direct login with username and password
+        Given I am logged in as '<user>' on '<env>'
+        And Created student 'John' 'Student' with email 'john.student@mail.tld'
+        And I am on the students management page
+        And Registering student 'John' with email 'john.student@mail.tld'
```

Comment:
Change the password during admin registration, so you don't have to set it later. If you change it later, the first login will be broken, because the firstLogin and forcePasswordChange flags at the same time. This leads to the bug you described in your code comment.